src/lib/y2storage/proposal/autoinst_devices_planner.rb

```diff
@@ -400,6 +415,24 @@ def remove_shadowed_subvols(planned_devices)
           end
         end
       end
+
+      DEFAULT_SIZES = {
+        "swap" => [DiskSize.MiB(512), DiskSize.GiB(2)].freeze
+      }.freeze
+
+      # @return [nil,Array<DiskSize>]
+      def auto_sizes_for(mount_point)
+        spec = volume_spec_for(mount_point)
+        return DEFAULT_SIZES[mount_point] if spec.nil?
+
+        default_min, default_max = DEFAULT_SIZES[mount_point] || []
+        [spec.min_size || default_min, spec.max_size || default_max]
+      end
+
+      def volume_spec_for(mount_point)
+        return nil if proposal_settings.volumes.empty?
```

Comment:
The code here expects `proposal_settings.format == :ng`. Is it not possible to use this with legacy format?